public/app/themes/justice/src/patch/js/ccfw-storage.js

```diff
@@ -1,29 +1,82 @@
 /**
  * A script for determining our storage method
  *
- * Falls back to cookies if localStorage is unavailable
+ * Falls back to cookies if localStorage is unavailable, otherwise
+ * we disable storage all together.
  */
-if (window.localStorage === undefined) {
-    Storage = {
-        getItem: (key) => {
-            const value = `; ${document.cookie}`;
-            const parts = value.split(`; ${key}=`);
+let Storage = {
+    disabled: true
+}
```

Comment:
By default, we start with a disabled Storage object. We can test directly against this.
The logic is that this object is overwritten if localStorage or Cookies are available.